pom.xml

```diff
@@ -265,7 +265,7 @@
 		<dependency>
 			<groupId>org.apache.httpcomponents</groupId>
 			<artifactId>httpclient</artifactId>
-			<version>4.3.1</version>
+			<version>4.5.1</version>
```

Comment:
I updated the httpclient version because I was having library version errors locally. Updating forced it to update to to the latest, and I figured since it worked, we may as well use the latest version.